network/ipmasq.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
```

Comment:
What about just naming the file _linux.go?

Reply:
In our next PR we are going to touch a bunch of files to get Flannel to compile on windows. If we follow this pattern all those files would get renamed to xxx_linux.go that feels very ugly and intrusive.... Can make the change if you think that it is more correct/idiomatic though.

Reply:
Also do you care about Nix's other than linux? Just wondering whether the _unspecified file is required in this case? For the larger change we've kept _unspecified for features that are not supported outside linux e.g. awsvpc_unspecfied, see in original PR:  https://github.com/coreos/flannel/pull/832/files#diff-c4be229b15064d7e338159bb6ef8cddf